src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let costPerDay = 40;
+  let totalCost = costPerDay * days;
+    
+    if (days >= 7) {
+      totalCost -= 50;
+    }
+    else if (days >= 3) {
+      totalCost -= 20;
+    }
+    
+  return totalCost;
```

Comment:
Fix indents.
Move discount values to the variables.
Also do not use else statement
![2023-01-24_20h16_24](https://user-images.githubusercontent.com/61904995/214375060-4da12db1-a947-4f6b-8cbb-1ed18968b797.png)